pom.xml

```diff
@@ -1277,7 +1277,7 @@
               <exclude>**/.settings/**/*</exclude>
 
               <!-- Proto/grpc generated wrappers -->
-              <exclude>**/sdks/python/apache_beam/runners/api/*.py</exclude>
+              <exclude>**/apache_beam/runners/api/*_pb2*.py</exclude>
```

Comment:
+1

I wanted to create apache_beam/core but we didn't have enough time before this release, I think we should wait a bit before doing any major refactoring at-least for one or two minor releases